services/api/mw2pdf/classes/PdfGenerator.js

```diff
@@ -9,6 +11,9 @@ import {
 } from 'pdf-lib'
 import { Pdf } from './Pdf.js'
 
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
This _seems_ like a really cool use of the `import` syntax but I've never seen it before. Would you be willing to add a quick pointer to some documentation for the next person who, like, is an idiot?

Reply:
I just yoinked it from the StackOverflow that I linked to in the commit message.

I'll actually include that link in the code as well since technically stackoverflow code is licensed under CC-SA-4.0

All that said I'm hoping we can delete this after (1) switching to babel and (2) changing the way we work with fonts in the first place.